For Ember Times No. 98: Add Ember Focus Trap + Contributors (#116)
ember-learn/ember-blog: source/2019-05-12-the-ember-times-issue-98.md

---

## [Intentional Element Focus with Ember Focus Trap 💥](https://twitter.com/josemarluedke/status/1127967059415953408)

> This week the **Global Accessibility Awareness Day (GAAD)** on May 16 inspired developers all around the world to reflect on the way they use and build applications for the web. In her [recent call-to-action for GAAD](https://blog.emberjs.com/2019/05/13/global-accessibility-awareness-day.html) 📣, [@melsumner](https://github.com/MelSumner) encouraged the Ember community to take the time to browse the web using only their keyboard and with the support of assistive technologies. Or to spend time to improve their own projects in regards to accessibility.

Reviewer: May I suggest the following for clarity?
```suggestion
On May 16, the **Global Accessibility Awareness Day (GAAD)** inspired developers all around the world to reflect on the way they use and build applications for the web. In her [recent call-to-action for GAAD](https://blog.emberjs.com/2019/05/13/global-accessibility-awareness-day.html) 📣, [@melsumner](https://github.com/MelSumner) encouraged the Ember community to take the time to browse the web using keyboard or an assistive technology, and to spend time to improve accessibility in their own projects.
```

Addon author [@josemarluedke](https://github.com/josemarluedke) followed this call-to-action and just published [Ember Focus Trap](https://github.com/josemarluedke/ember-focus-trap)! It's based on the JavaScript library [Focus Trap](https://github.com/davidtheclark/focus-trap) that helps you to intentionally trap focus in certain DOM nodes. This makes it easy for you to improve the keyboard accessibility of otherwise inaccessible elements such as modal dialogs.

> And with **Ember Focus Trap** adding focus to interactive elements in your Ember app is only an `ember install` away. Check it [out today](https://josemarluedke.github.io/ember-focus-trap/)!

Reviewer: ```suggestion
With **Ember Focus Trap**, adding focus to interactive elements in your Ember app is only an `ember install` away. Check it [out today](https://josemarluedke.github.io/ember-focus-trap/)!
```
